### PR TITLE
Rewrite tests to Page Object Model

### PR DESCRIPTION
**Pull request by kravciak**
Initial rewrite of tests to POM.
- added rancher UI helpers to work with custom elements (checkboxes, ...)
- split installation tests and basic landing page checks
- added rancher-common helpers (adding repos, installing apps, login, ...)
- multiple test stability & readability improvements

### CODE REVIEW

**Review by viccuad — APPROVED** [review]
LGTM from the test side point (I don't have anything to add from the Typescript point)

**Review by jordojordo — CHANGES_REQUESTED** [review]
One minor change, and a few suggestions - Otherwise looks good

File: tests/playwright.config.ts [hunk 1]
```diff
@@ -16,7 +16,7 @@ export default defineConfig({
   /* Retry on CI only */
   retries: process.env.CI ? 1 : 0,
   /* Opt out of parallel tests on CI. */
-  workers: process.env.CI ? 1 : undefined,
+  workers: process.env.CI ? 1 : 1,
```
[the remark this answers is not in the document]
Comment (kravciak) [reply tests/playwright.config.ts]: fixed

File: tests/e2e/exec.spec.ts [hunk 2]
```diff
@@ -0,0 +1,26 @@
+import { test } from './rancher-test'
+import { RancherCommonPage } from './pages/rancher-common.page';
+
+const charts = [
+  {name: 'Jaeger Operator', namespace: 'jaeger', chart: "jaeger-operator"},
+  {name: 'Monitoring', project: '(None)', chart: "rancher-monitoring"},
+  {name: 'Kubewarden', project: 'Default', chart: "rancher-kubewarden-controller"},
+]
```
[the remark this answers is not in the document]
Comment (kravciak) [reply tests/e2e/exec.spec.ts]: I added another commit fixing this, thanks for the suggestion!

**Review by jordojordo — APPROVED** [review]
Looks good!

**Pull request merged**